AdSecGH/AdSecGHInfo.cs

```diff
@@ -142,112 +129,88 @@ private static string TryFindPluginPath(string keyword)
+    public override string AuthorName {
+      get {
+        //Return a string identifying you or your company.
+        return Company;
       }
     }
-    public override string Description
-    {
-      get
-      {
+    public override string Description {
+      get {
         //Return a short string describing the purpose of this GHA library.
         return "Official Oasys AdSec Grasshopper Plugin" + Environment.NewLine
           + (isBeta ? Disclaimer : "")
         + Environment.NewLine + "The plugin requires an AdSec 10 license to load. "
         + Environment.NewLine + "Contact oasys@arup.com to request a free trial version. "
         + System.Environment.NewLine + TermsConditions
         + System.Environment.NewLine + Copyright;
-
       }
     }
-    public override Guid Id
-    {
-      get
-      {
-        return GUID;
+    public override Bitmap Icon {
+      get {
+        //Return a 24x24 pixel bitmap to represent this GHA library.
+        return null;
       }
     }
-
-    public override string AuthorName
-    {
-      get
-      {
-        //Return a string identifying you or your company.
-        return Company;
+    public string icon_url {
+      get {
+        return "https://raw.githubusercontent.com/arup-group/GSA-Grasshopper/main/Documentation/GettingStartedGuide/Icons/GsaGhLogo.jpg";
       }
     }
-    public override string AuthorContact
-    {
-      get
-      {
-        //Return a string representing your preferred contact details.
-        return Contact;
+    public override Guid Id {
+      get {
+        return GUID;
       }
     }
-    public override string Version
-    {
-      get
-      {
+    public override string Name {
+      get {
+        return ProductName;
+      }
+    }
+    public override string Version {
+      get {
         if (isBeta)
           return Vers + "-beta";
         else
           return Vers;
       }
     }
+    internal const string Company = "Oasys";
+    internal const string Contact = "https://www.oasys-software.com/";
+    internal const string Copyright = "Copyright © Oasys 1985 - 2022";
+    internal const string PluginName = "AdSecGH";
+    internal const string ProductName = "AdSec";
+    internal const string TermsConditions = "Oasys terms and conditions apply. See https://www.oasys-software.com/terms-conditions for details. ";
+    internal const string Vers = "0.9.17";
+    internal static string Disclaimer = PluginName + " is pre-release and under active development, including further testing to be undertaken. It is provided \"as-is\" and you bear the risk of using it. Future versions may contain breaking changes. Any files, results, or other types of output information created using " + PluginName + " should not be relied upon without thorough and independent checking. ";
+    internal static Guid GUID = new Guid("f815c29a-e1eb-4ca6-9e56-0554777ff9c9");
+    internal static bool isBeta = true;
+  }
+
+  internal sealed class PluginInfo {
```

Comment:
nested classes shouldn't be at the beginning?

Reply:
it´s not nested ;)

Reply:
we should probably make it it´s own file..